External/Plugins/ProjectManager/Projects/Haxe/HaxeProject.cs

```diff
@@ -436,11 +458,39 @@ private void ParseHxmlEntries(string[] lines, List<string> defs, List<string> cp
                 }
                 else if (!trimmedLine.StartsWith("#") && trimmedLine.EndsWith(".hxml", StringComparison.OrdinalIgnoreCase))
                 {
-                    string subhxml = this.GetAbsolutePath(trimmedLine);
-                    if (File.Exists(subhxml))
+                    var subhxml = GetAbsolutePath(trimmedLine);
```

Comment:
In case of recursive hxmls in different sub folders, should the function be provided a reference path?